docs/schema-loader.md

```diff
@@ -429,15 +429,15 @@ By default, the schema loader enables auto-scaling of RU for all tables: RU is s
 
 Here are the supported data types in Scalar DB and their mapping to the data types of other databases.
 
-| Scalar DB | Cassandra | Cosmos DB      | DynamoDB | MySQL    | PostgreSQL       | Oracle         | SQL Server      |
-|-----------|-----------|----------------|----------|----------|------------------|----------------|-----------------|
-| BOOLEAN   | boolean   | boolean (JSON) | BOOL     | boolean  | boolean          | number(1)      | bit             |
-| INT       | int       | number (JSON)  | N        | int      | int              | int            | int             |
-| BIGINT    | bigint    | number (JSON)  | N        | bigint   | bigint           | number(19)     | bigint          |
-| FLOAT     | float     | number (JSON)  | N        | double   | float            | binary_float   | float(24)       |
-| DOUBLE    | double    | number (JSON)  | N        | double   | double precision | binary_double  | float           |
-| TEXT      | text      | string (JSON)  | S        | longtext | text             | varchar2(4000) | varchar(8000)   |
-| BLOB      | blob      | string (JSON)  | B        | longblob | bytea            | RAW(2000)      | varbinary(8000) |
+| Scalar DB | Cassandra | Cosmos DB      | DynamoDB | MySQL    | PostgreSQL       | Oracle         | SQL Server      | SQLite  |
+|-----------|-----------|----------------|----------|----------|------------------|----------------|-----------------|---------|
+| BOOLEAN   | boolean   | boolean (JSON) | BOOL     | boolean  | boolean          | number(1)      | bit             | boolean |
+| INT       | int       | number (JSON)  | N        | int      | int              | int            | int             | int     |
+| BIGINT    | bigint    | number (JSON)  | N        | bigint   | bigint           | number(19)     | bigint          | bigint  |
```

Comment:
It seems SQLite3 has 5 Type Affinities https://www.sqlite.org/datatype3.html and `bigint` and `float` are interpreted as one of the type affinities. Using the type affinity name is a bit more correct?

Reply:
Ah, you've already mentioned the page above. Thanks. Noticed Column Data type (INTEGER, REAL, TEXT, BLOB) might be better.

Reply:
I understand what you say.

On the other hand, we can even state all types are mapped to INTEGER because SQLite in reality has type for each value (not for column)

<img width="489" alt="image" src="https://user-images.githubusercontent.com/498788/209223009-c1cd08bc-230e-4df6-acef-bf358f713ab9.png">

(https://www.sqlite.org/flextypegood.html).

Therefore, we may choose the one from the following (and I feel 1. is the best).

1. Use the same type names as the Scalar DB
2. Use the storage classes (INTEGER, REAL, TEXT, or BLOB)
3. Use only INTEGER (should be the worst but possible choice)

Reply:
Thanks for the explanation! That makes sense.